strawberry/field.py

```diff
@@ -40,13 +40,15 @@ def __init__(
         resolver=None,
         name=None,
         description=None,
+        permission_classes=None
     ):
         self._wrapped_obj = obj
         self.is_subscription = is_subscription
         self.is_input = is_input
         self.field_name = name
         self.field_resolver = resolver
         self.field_description = description
+        self.permission_classes = permission_classes
```

Comment:
```suggestion
        self.permission_classes = permission_classes or []
```

if we default it to a list we can skip some ifs below :)